dir/sample.txt

```diff
@@ -1,4 +1,4 @@
-This is sample text.
+his is sample text.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/litmon/danger-sample/blob/defaf9cb246539d039dd2d67dd92c454275040b7/dir/sample.txt#L1"></span>It is typo</td>
    </tr>
  </tbody>
</table>